### ISSUE TITLE
Uso de DISTINCT 

### ISSUE THREAD

**Issue by IgnaciaJ** [IIC2413/Syllabus-2021-1#97]
Hola! Quería saber si para hacer las consultas en álgebra relacional podemos usar DISTINCT(delta)? 
Gracias!

**Comment by Skablop** [comment]
En álgebra relacional no existe un operador distinct, pero recuerda que el operador de proyección no permite duplicados entre las filas.

**Comment by IgnaciaJ** [comment]
Gracias!! y si para las direcciones escribimos una calle, Comuna, y nos piden todas las unidades de San Joaquin, existe algún equivalente del LIKE %%, para hacerlo en álgebra relacional? O cambiamos nuestras direcciones a poner solo una comuna?

**Comment by Skablop** [comment]
No entiendo muy bien tu pregunta, pero en álgebra relacional no hay un equivalente al LIKE. Sobre lo otro, se supone que cada dirección está asociada solo a una comuna. 


Además, no debería ser necesario usar LIKE ya que los nombres de las comunas no aparecen escritos de manera diferente (no deberías tener "SAN JOAQUIN" y "San Joaquin" a la vez, por ejemplo).

**Comment by IgnaciaJ** [comment]
sisi pero entonces al final la dirección de las unidades era poner solo una comuna y no una calle asociada? gracias!

**Comment by Skablop** [comment]
No, cada dirección debe tener asociada una comuna, por lo que preguntar por una comuna o una dirección específica es diferente. Así, tus unidades deben tener como atributos su dirección y la comuna en la que se encuentra esa dirección.

**Comment by IgnaciaJ** [comment]
buenisimo gracias!
